Math/sqrt(x).js

```diff
@@ -0,0 +1,20 @@
+/* 
```

Comment:
This is a fairly simple program, and you're using Math.floor(Math.sqrt(x)) inbuilt function, so no need to add these type of problems unless you're implementing sqrt yourself